### PR TITLE
sw/adding a new ucr for lactating women

### PR DESCRIPTION
**Pull request by SunainaWalia**
@mkangia, we created a new dynamic report for lactating women which needs to be made static.

The relevant Asana is linked here: https://app.asana.com/0/1173592047649882/1174735811154209
The link to the dynamic report is here: https://india.commcarehq.org/a/sunaina-sandbox/configurable_reports/reports/edit/0bf488e5a1e24a09c449ba35c84ca7cb/

### CODE REVIEW

File: custom/icds_reports/ucr/reports/ls/ls_v2_report_lactating_women_names [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    "reach-test",
+    "icds-test",
+    "icds-sql",
+    "icds-cas",
+    "cas-lab",
+    "icds-cas-sandbox"
+  ],
+  "server_environment": [
+    "india",
+    "icds"
```
Comment (mkangia) [inline custom/icds_reports/ucr/reports/ls/ls_v2_report_lactating_women_names:26]: I think we should add "icds-staging" as well now for icds environments.

**Review by mkangia — COMMENTED** [review]
@SunainaWalia 
is this copied over somewhere and then edited or is entirely new?

**Comment by SunainaWalia** [comment]
> @SunainaWalia
> is this copied over somewhere and then edited or is entirely new?

This is a new report. But the configuration is very similar to ls_v2_report_pregnant_women_names. I copied over the domain and environment info from that report and edited the agg columns, filters and columns as needed.

File: custom/icds_reports/ucr/reports/ls/ls_v2_report_lactating_women_names (hunk 1 above)
Comment (SunainaWalia) [reply custom/icds_reports/ucr/reports/ls/ls_v2_report_lactating_women_names]: good point. added this.

**Review by mkangia — APPROVED** [review]
Cool, thanks @SunainaWalia ..
I trust that you already tested the report.. syntax wise looks good.
Would be good to get another approval from @Rohit25negi  as well.

**Comment by SunainaWalia** [comment]
> Cool, thanks @SunainaWalia ..
> I trust that you already tested the report.. syntax wise looks good.
> Would be good to get another approval from @Rohit25negi as well.

Thanks @mkangia ! I did test the report, here's the link to the dynamic UCR in case you'd like to have a look.

https://india.commcarehq.org/a/sunaina-sandbox/configurable_reports/reports/edit/0bf488e5a1e24a09c449ba35c84ca7cb/#

@Rohit25negi, like @mkangia suggests, could you review this as well and let me know if anything seems off or needs to be changed? thank you!

**Review by Rohit25negi — APPROVED** [review]

**Comment by mkangia** [comment]
Have pinged @SunainaWalia to update the filename to add ".json" .

**Comment by SunainaWalia** [comment]
> Have pinged @SunainaWalia to update the filename to add ".json" .
added this @mkangia

**Pull request merged**